checkbox-ng/checkbox_ng/config.py

```diff
@@ -33,51 +33,98 @@
 _logger = logging.getLogger("config")
 
 
+# The order here defines the priority
+# launcher > ~/.config > /etc/xdg > $SNAP_DATA
 SEARCH_DIRS = [
-        '$SNAP_DATA',
-        '/etc/xdg/',
-        '~/.config/',
-    ]
+    "~/.config/",
+    "/etc/xdg/",
+    "$SNAP_DATA",
+]
 
 
 def expand_all(path):
     """Expand both: envvars and ~ in `path`."""
     return os.path.expandvars(os.path.expanduser(path))
 
 
+def _search_configs_by_name(name: str) -> list[str]:
+    """
+    Returns all well known config locations that have a `name` file
+    in them
+    """
+    if os.path.isabs(expand_all(name)):
+        return [name]
+    to_r = []
+    for sdir in SEARCH_DIRS:
+        config = expand_all(os.path.join(sdir, name))
+        if os.path.exists(config):
+            to_r.append(config)
+        else:
+            _logger.info("Referenced config file doesn't exist: %s", config)
```

Comment:
We shouldn't INFO about that, maybe DEBUG worth, but even that is a stretch. And the message here is quite strong for a typical case, even more so considering there's no message before the loop.
For instance the following is much less scary:
```
DEBUG: searching for checkbox.conf files...
DEBUG: not found in ~/.config
...
```